console-subscriber/examples/grpc_web.rs

```diff
@@ -0,0 +1,165 @@
+//! Example of using the console subscriber with tonic-web.
+//! This example requires the `grpc-web` feature to be enabled.
+//! Run with:
+//! ```sh
+//! cargo run --example grpc_web --features grpc-web
+//! ```
+use std::time::Duration;
+
+use console_subscriber::ConsoleLayer;
+
+static HELP: &str = r#"
+Example console-instrumented app
```

Comment:
Can we update this short description to mentioned grpc_web as well.

Reply:
I will update it later.